src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
```

Comment:
Consider extracting the magic number '40' to a constant variable named 'dailyRentalCost', for better code readability and maintainability. 😊